content/en/_index.html

```diff
@@ -8,9 +8,9 @@
 
 {{< site-searchbar >}}
 
-{{< blocks/section id="oceanNodes" >}}
-{{% blocks/feature image="flower" %}}
-[Kubernetes]({{< relref "/docs/concepts/overview/" >}}), also known as K8s, is an open source system for automating deployment, scaling, and management of containerized applications.
+{{< blocks/section class="k8s-overview" >}}
+{{% blocks/feature image="flower" id="feature-primary" %}}
+[Kubernetes]({{< relref "/docs/concepts/overview/" >}}), also known as K8s, is an open-source system for automating deployment, scaling, and management of containerized applications.
```

Comment:
```suggestion
[Kubernetes]({{< relref "/docs/concepts/overview/" >}}), also known as K8s, is an open source system for automating deployment, scaling, and management of containerized applications.
```

ref: https://github.com/cncf/foundation/blob/main/style-guide.md#1-cloud-native-and-open-source

Reply:
We also added this to our own word list a little while back! https://kubernetes.io/docs/contribute/style/style-guide/#kubernetes-io-word-list

Reply:
I'll omit the whitespace change here.